t/00-compile-check-all.t

```diff
@@ -16,33 +16,21 @@
 
 use strict;
 use warnings;
-use Test::Compile;
 # We need :no_end_test here because otherwise it would output a no warnings
 # test for each of the modules, but with the same test number
 use Test::Warnings ':no_end_test';
-
 use Cwd;
 
 use FindBin '$Bin';
 use lib "$Bin/..";
 
-use cv;
-
-chdir "$Bin/..";
-cv::init;
-
-my $test = Test::Compile->new();
-
-my @files = $test->all_pm_files(".");
-
-for my $file (@files) {
-    next if ($file =~ /main.pm/);
-    $file =~ s,^\./,,;
-    $test->ok($test->pm_file_compiles($file), "Compile test for $file");
-}
+use Test::Strict;
 
-@files = ('isotovideo', $test->all_pl_files("."));
-for my $file (@files) {
-    $test->ok($test->pl_file_compiles($file), "Compile test for $file");
-}
-$test->done_testing();
+$Test::Strict::TEST_SYNTAX   = 1;
+$Test::Strict::TEST_STRICT   = 1;
+$Test::Strict::TEST_WARNINGS = 1;
+$Test::Strict::TEST_SKIP     = [
+    't/data/tests/main.pm',
```

Comment:
Seems like it is executed within `t` so the relative path would just be `data/tests/main.pm`.

Reply:
Yeah, the problem is that the `chdir "$Bin/..";` has been removed

Reply:
done